happi/cli.py

```diff
@@ -377,10 +378,10 @@ def load(ctx, item_names: List[str]):
 def update(ctx, json_data: str):
     """Update happi db with JSON_DATA payload"""
     # retrieve client
-    print(json_data)
     client = ctx.obj
     if len(json_data) < 1:
-        raise click.Abort()
+        raise click.BadArgumentUsage('No json data given')
+    print(json_data)
```

Comment:
Should this even be here? Seems like a potential mistake...
```suggestion
```

Reply:
I just moved it from line 380, but it looks like a debug print to me

Reply:
@tangkong? Or was this brought in from the original pre-click implementation?

Reply:
Erm, whoops this looks like my fault.  Good catch! 😬   I don't see it in the pre-click implementation, and I can see myself forgetting that there since I didn't spend a lot of time in that subcommand